lib/engine/step/g_18_zoo/buy_sell_par_shares.rb

```diff
@@ -27,6 +34,33 @@ def process_buy_company(action)
           super
 
           @game.available_companies.delete(action.company)
+          @game.apply_custom_ability(action.company)
+        end
+
+        def get_par_prices(_entity, _corp)
+          super
```

Comment:
one pass

super.reject do
 .. && ..

end

Reply:
done in 4800b1d3